client/new-stack-editor/lib/controllers/base.coffee

```diff
@@ -0,0 +1,10 @@
+kd = require 'kd'
+
+
+module.exports = class BaseController extends kd.Object
+
+  constructor: (options = {}, data) ->
+
+    super options, data
+
+    @editor = @getOption 'editor'
```

Comment:
in this file I may re-set the option and delete editor from there after I've assigned it to local variable. so, there are some plans on `@editor` variable for the future.